source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -608,17 +613,20 @@ def findOverlayClasses(self,clsList):
 			if clsList[0]==IAccessible and len(clsList)==3 and self.IAccessibleRole==oleacc.ROLE_SYSTEM_CLIENT and self.childCount==0:
 				clsList.insert(0,ContentGenericClient)
 
-	def __init__(self,windowHandle=None,IAccessibleObject=None,IAccessibleChildID=None,event_windowHandle=None,event_objectID=None,event_childID=None):
+	# C901: 'IAccessible.__init__' is too complex
+	def __init__(  # noqa: C901
+			self,
+			windowHandle: Optional[int] = None,
+			IAccessibleObject: Optional[Union[IUnknown, IA.IAccessible, IA2.IAccessible2]] = None,
+			IAccessibleChildID: Optional[int] = None,
+			event_windowHandle: Optional = None,
+			event_objectID: Optional = None,
+			event_childID: Optional = None

```

Comment:
Based on the doc strings and context I assume these are `Optional[int]`

```suggestion
			event_windowHandle: Optional[int] = None,
			event_objectID: Optional[int] = None,
			event_childID: Optional[int] = None
```

Reply:
I would assume so to, but not confidently. I wouldn't want to risk misleading future developers who might rely on on this.